app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -183,30 +183,30 @@
 
 .gem-c-image-card__context,
 .gem-c-image-card__metadata {
-  @include govuk-font($size: false);
   font-size: 16px;
   font-size: govuk-px-to-rem(16px);
   margin: 0 0 calc(govuk-spacing(3) / 2);
   color: govuk-colour("dark-grey");
+  @include govuk-font($size: false);
```

Comment:
Unrelated to your change, but strange - the font size is set twice, then set to false in the mixin. I might circle back and check this one later, just making a note.

Reply:
My understanding is that it wants everything from the govuk-font mixin except the size.

If it helps, this is the resulting rule:

```
.gem-c-image-card__context,.gem-c-image-card__metadata {
    font-size: 16px;
    font-size: 1rem;
    margin: 0 0 7.5px;
    color: #505a5f;
    font-family: "GDS Transport",arial,sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    font-weight: 400
}
```

which doesn't look like anything funky is going on.